scripts/run_lintrunner.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""
+Pre‑push hook wrapper for Lintrunner.
+
+✓ Stores a hash of .lintrunner.toml in the venv
+✓ Re-runs `lintrunner init` if that file's hash changes
+"""
+
+from __future__ import annotations
+
+import hashlib
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+
+REPO_ROOT = Path(__file__).resolve().parents[1]
+LINTRUNNER_TOML_PATH = REPO_ROOT / ".lintrunner.toml"
+
+# This is the path to the pre-commit-managed venv
+VENV_ROOT = Path(sys.executable).parent.parent
```

Comment:
Is our expectation that this is related to the `python` interpreter we're running this file with?

i.e.:

```
/home/myuser/.venv/bin/python/../../
```

If a user runs this with a system level python this could produce weird results

```
/usr/bin/local/python/../../
```

Reply:
Yeah, it's specifically supposed to be relative to the python interpreter running inside the pre-commit virutal environment, but should work with any virtual environment.

Good call that this will cause bad behavior if run from a system level python. I'll add a fix for that